### ISSUE TITLE
Always the same bounding box predicted 

### ISSUE THREAD

**Issue by AntoineMath** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#34]
I modified your code to use 2 channels depth-thermal image.
I have 800 training data of different body positions.
But after a decent training, the predicted bbox are the same, no matter the body position in the picture. Exact same coordinates, exact same scores. (I use top_k=1 since I have only 1 object per image)
![Capture du 2019-12-12 14-40-56](https://user-images.githubusercontent.com/44801785/70716836-6d039680-1ced-11ea-8319-2d414afa5624.png)
Same with top_k = 200 :
![Capture du 2019-12-12 14-47-11](https://user-images.githubusercontent.com/44801785/70717291-56117400-1cee-11ea-84a8-14942d3bca5d.png)
![Capture du 2019-12-12 14-41-49](https://user-images.githubusercontent.com/44801785/70716886-8573b100-1ced-11ea-8c7b-d88128548364.png)

I do not use data augmentation, but I think this is another problem.
It's like the loss was trapped in a local minimum, but idk.
Did you have the same problem ?

EDIT: I figure out my predictions are based on the number of each body position images :
left, dataset : 56 standing, 53 sitting, 73 lying down
right, dataset : 56 standing, 108 sitting, 73 lying down
![detection_thermique](https://user-images.githubusercontent.com/44801785/70983290-c63a4400-20b8-11ea-8909-e0d1ae209767.png)

this is just a small dataset to perform tests... but I dont understand why the loss is stucked here...

**Comment by AntoineMath** [comment]
Solved : I did not used data augmentation. And the moment I tried to use it, it worked. I'm surprised that it had a so huge impact.

**Comment by AntoineMath** [comment]
Solved : I did not used data augmentation. And the moment I tried to use it, it worked. I'm surprised that it had a so huge impact.
